.github/actions/generate-release-notes/releaseNotes.template.md

```diff
@@ -0,0 +1,6 @@
+Today we are releasing the ${buildDescription} of the `dotnet monitor` tool. This release includes:
```

Comment:
I think this [workflow](https://github.com/dotnet/dotnet-monitor/blob/main/.github/workflows/add-markdown-feedback.yml#L28) will end up considering this template markdown file in the future if we need to make changes to it. Can you exclude this, if I'm reading it correctly?